db/migrate/20171120161420_add_base_info_to_pull_request.rb

```diff
@@ -0,0 +1,7 @@
+class AddBaseInfoToPullRequest < ActiveRecord::Migration[5.1]
+  def change
+    add_column :pull_requests, :base_ref, :string
```

Comment:
Since shas have a specific length, might as well limit it to 40 here.

Reply:
The base ref is not a sha, it's (always?) the merge base branch name. I don't think git enforces a length limit on branches.

Reply:
My bad. In this case, we should wonder if 255 chars is enough. If we bump that limit a tad higher, say 1024, I think it won't be worth handling overflows.

Reply:
Sanity: Do I just need to add `limit: 1024`?

Reply:
Yes